src/plugins/grid_3d/Grid3D.qml

```diff
@@ -175,107 +175,36 @@ GridLayout {
   }
 
   Text {
-    Layout.columnSpan: 2
+    Layout.columnSpan: 1
```

Comment:
If we keep this at 2, then I don't think we need the dummy item below for formatting
```suggestion
    Layout.columnSpan: 2
```